Cargo.toml

```diff
@@ -29,7 +29,13 @@ rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
 
 [features]
 # Default features.
-default = ["colors", "widgets"]
+default = ["colors", "widgets", "track_change_detection"]
```

Comment:
Change detection tracking is a debugging feature, it shouldn't be enabled by default.